programs/zero_matrix.py

```diff
@@ -0,0 +1,39 @@
+# when find a cell with 0, convert all de column and all de row to 0s
+
+
+def zero(a):
+    # seek a cell with 0 and storage the column in a list and de row in a list
+    cols = []
+    rows = []
+    for row in range(len(a)):
+        for col in range(len(a[row])):
+            if a[row][col] == 0:
+                if col not in cols:
```

Comment:
If you use a set instead of a list, there is no need to check if col is in cols. 
Remember sets are better for checking membership.

https://docs.python.org/2/library/stdtypes.html#set-types-set-frozenset